components/dropdown/demo/dropdown-menu.html

```diff
@@ -97,6 +97,37 @@ <h2>Normal</h2>
 			</template>
 		</d2l-demo-snippet>
 
+		<h2>With Header/Footer</h2>
+		<d2l-demo-snippet>
+			<template>
+				<d2l-dropdown>
+					<button class="d2l-dropdown-opener">Open it!</button>
+					<d2l-dropdown-menu id="dropdown" no-padding-footer>
+						<div slot="header" style="font-weight: bold; width: 100%; text-align: center;">Physics Topics</div>
+						<d2l-menu>
+							<d2l-menu-item text="Introduction" id="first-item"></d2l-menu-item>
```

Comment:
Are the `first-item` and `last-item` ids used at all?

Reply:
It doesn't look like it, actually. It's in the demo page and the code examples in the `dropdown-menu` `readme.md` but the only place I see any use is in a [test for `dropdown-menu`](https://github.com/BrightspaceUI/core/blob/master/components/dropdown/test/dropdown-menu.test.js#L51) but that's only `first-item` and not `last-item`.

Reply:
Checked with Allan, they were back in the Polymer tests but aren't used apart from the test-case, so all except those test-cases have been removed.